scripts/run_driver_tests.psm1

```diff
@@ -273,6 +273,7 @@ function Invoke-Test
 
     # Execute Test.
     Write-Log "Executing $TestName $TestArgs"
+
```

Comment:
This blank line doesn't appear useful, and it separates the comment (line 274) from the code that actually does what the comment says.  Recommend reverting this file.